src/System.IO.FileSystem/src/System/IO/WinRTFileSystem.cs

```diff
@@ -725,6 +725,11 @@ public override void SetLastWriteTime(string fullPath, DateTimeOffset time, bool
             // "System.DateModified" property is readonly
         }
 
+        public override string[] GetLogicalDrives()
+        {
+            return DriveInfoInternal.GetLogicalDrives();
```

Comment:
Similarly here: we should just make this ```throw new PlatformNotSupportedException();``` and get rid of the file containing an implementation of GetLogicalDrives that does the same.  The only DriveInfoInternal.GetLogicalDrives would then be one used by Win32, and it only exists because it's more than a couple of lines and is used by multiple assemblies.